home/jobs/OMERO-docs/config.xml

```diff
@@ -17,7 +17,7 @@
         <hudson.model.StringParameterDefinition>
           <name>MERGE_COMMAND</name>
           <description></description>
-          <defaultValue>merge SPACENAME --no-ask --reset</defaultValue>
+          <defaultValue>merge develop --no-ask --reset</defaultValue>
```

Comment:
Is this intentional?

Reply:
Changed back to `SPACEBRANCH` to match the Git configuration